src/rest-server/src/models/v2/virtual-cluster/k8s.js

```diff
@@ -32,6 +32,26 @@ const vcData = {
 };
 
 
+const fetchNodes = async () => {
+  const nodes = await axios({
+    method: 'get',
+    url: `${kubernetesConfig.apiserver.uri}/api/v1/nodes`,
+  });
+  return nodes.data.items.filter((node) => {
+    if (node.metadata.labels['pai-worker'] !== 'true') {
+      return false;
+    }
+    // check node readiness
+    for (let i = node.status.conditions.length - 1; i >= 0; i --) {
+      const condition = node.status.conditions[i];
+      if (condition.type === 'Ready' && condition.status === 'True') {
+        return true;
+      }
```

Comment:
Here we only count the node which condition is ready. How about the node in other condition, such as `MemoryPressure`, `OutOfDisk`. Refer to https://kubernetes.io/docs/concepts/architecture/nodes/#condition
Can we just filter out the nodes which 
```js
condition.type === 'Ready' && (condition.status === 'False' || condition.status === 'Unknown')
```